tools/cli/adminKafkaCommands.go

```diff
@@ -479,11 +479,6 @@ func doRereplicate(
 
 // AdminRereplicate parses will re-publish replication tasks to topic
 func AdminRereplicate(c *cli.Context) {
-	numberOfShards := c.Int(FlagNumberOfShards)
```

Comment:
maybe make it optional (default=16k) here https://github.com/uber/cadence/blob/af307533b00b050cf62ced0f788697328ec5ac14/tools/cli/adminCommands.go#L395-L394

Reply:
I think we should leave it explicit. If user configures cluster with different number of shards, such silent default can result in very unexpected results.